util/envDynawo.sh

```diff
@@ -1592,6 +1592,9 @@ deploy_dynawo() {
   cp -P $DYNAWO_LIBXML_HOME/lib/*.* lib/
   echo "deploying libiidm libraries"
   cp -P $DYNAWO_LIBIIDM_HOME/lib/*.* lib/
+  if [ -d "$DYNAWO_LIBIIDM_HOME/bin" ]; then
+    cp -n -R -P $DYNAWO_LIBIIDM_HOME/bin/* bin/
```

Comment:
I think we should ensure the bin target directory exists before trying to copy: `mkdir -p bin`
We could try-create only here if the `LIBIIDM_HOME` contains a `bin/`. Do you agree @gautierbureau ?

Reply:
yes you are right it would be better to do : [ -d "$DYNAWO_LIBIIDM_HOME/bin" ] && cp -n -R -P $DYNAWO_LIBIIDM_HOME/bin/* bin/